packages/syntax-icons/src/Icons.stories.tsx

```diff
@@ -0,0 +1,370 @@
+  { name: "Globe", component: Globe },
+  { name: "Graphs", component: Graphs },
+  { name: "Hand", component: Hand },
+  { name: "HangUp", component: HangUp },
+  { name: "HeartFilled", component: HeartFilled },
+  { name: "HeartUnfilled", component: HeartUnfilled },
+  { name: "Help", component: Help },
+  { name: "Home", component: Home },
+  { name: "IdentifyDocument", component: IdentifyDocument },
+  { name: "Information", component: Information },
+  { name: "Joystick", component: Joystick },
+  { name: "Keyboard", component: Keyboard },
+  { name: "Leaf", component: Leaf },
+  { name: "Lightbulb", component: Lightbulb },
+  { name: "List", component: List },
+  { name: "Location", component: Location },
+  { name: "Menu", component: Menu },
+  { name: "Message", component: Message },
+  { name: "MicrophoneOff", component: MicrophoneOff },
+  { name: "MicrophoneOn", component: MicrophoneOn },
+  { name: "More", component: More },
+  { name: "MultiMediaDocument", component: MultiMediaDocument },
+  { name: "MultipleUsers", component: MultipleUsers },
+  { name: "NewStarter", component: NewStarter },
+  { name: "Pause", component: Pause },
+  { name: "Pencil", component: Pencil },
+  { name: "PhoneOff", component: PhoneOff },
+  { name: "PhoneOn", component: PhoneOn },
+  { name: "Play", component: Play },
+  { name: "Progress", component: Progress },
+  { name: "Refresh", component: Refresh },
+  { name: "Safety", component: Safety },
+  { name: "Screen", component: Screen },
+  { name: "Search", component: Search },
+  { name: "SearchUsers", component: SearchUsers },
+  { name: "Send", component: Send },
+  { name: "Settings", component: Settings },
+  { name: "Shop", component: Shop },
+  { name: "Shuffle", component: Shuffle },
+  { name: "SideArrow", component: SideArrow },
+  { name: "SquareEnclosedPlus", component: SquareEnclosedPlus },
+  { name: "StarFilled", component: StarFilled },
+  { name: "StarHalfFilled", component: StarHalfFilled },
+  { name: "Stars", component: Stars },
+  { name: "StarUnfilled", component: StarUnfilled },
+  { name: "Subtitles", component: Subtitles },
+  { name: "Suitcase", component: Suitcase },
+  { name: "SuperTutor", component: SuperTutor },
+  { name: "Sync", component: Sync },
+  { name: "TeachingCertificate", component: TeachingCertificate },
+  { name: "TextDocument", component: TextDocument },
+  { name: "ThumbsDown", component: ThumbsDown },
+  { name: "ThumbsUp", component: ThumbsUp },
+  { name: "Timer", component: Timer },
+  { name: "ToggleSettings", component: ToggleSettings },
+  { name: "Translate", component: Translate },
+  { name: "TwoUsers", component: TwoUsers },
+  { name: "Type", component: Type },
+  { name: "User", component: User },
+  { name: "Wifi", component: Wifi },
+  { name: "WifiRouter", component: WifiRouter },
+];
+
+const MUIIcons = [{ name: "Repeat", component: Repeat }];
+
+export default {
+  title: "Icons/Icons",
```

Comment:
@somethiiing should we move this to `Components/Icon` so it shows up in the Syntax core list? Since `Icon` is now part of syntax core
```suggestion
  title: "Components/Icon",
```

Reply:
per private convo, we'll move it to the top next to "colors" since i think Icons is its own "library"